### PR TITLE
Bug 1227537 - [TV][2.5] Failed to navigate back

### PR DESCRIPTION
**Pull request by rickychien**
https://bugzilla.mozilla.org/show_bug.cgi?id=1227537

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4eac6c7b6830626facdc271e9480c798797d79d4

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3940108411e95e63c617d9ea8b50e48607088498

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d36e4103eb0b92b5fefeafa89e3bf67f86095307

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=21dbc20c26ca92bcd223db31ef770e4193884908

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3b3804ad93b1ece7920cf0f37efe18ed2e7c1a89

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c44bab0e2ee4a13599cdbb481dfa17a79167c25e

File: tv_apps/smart-system/js/child_window_factory.js [hunk 1]
```diff
@@ -60,12 +60,12 @@
         return;
       }
 
-      // <a href="" target="_blank"> should never be part of the app
+      // <a href="" target="_blank|_new"> should be part of the app
       // except while FTU is running: windows must be closable & parented by FTU
-      if (evt.detail.name == '_blank' &&
+      if ((evt.detail.name === '_blank' || evt.detail.name === '_new') &&
```
Comment (luke-chang) [inline tv_apps/smart-system/js/child_window_factory.js]: We need to consider what it should do when the `name` isn't specified.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=956fe2efbaffe27f3e156762008daa55eceb3782

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=75b5b5f2823d5d91eb4d8688473b8779a9e6a606

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c2e7de9e6f3aa80db9d30369c04fc4f1768f007c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=46e8fadabae1585a2b08af6419a7ab0e5c869a65

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=38fac3c4435e8bda3b3d8013c9aebe64b7c08f99

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a4aff65120108b73cd0a055409151f09cbbdb1ac

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=39c229a3c7520394e766cbc7b85d6973aace2181

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9c33f84c9efb6dd773296507bd73a81aac4703c8

**Pull request merged**